### PR TITLE
Added `Warhead::Status`

### PR DESCRIPTION
**Pull request by Michal78900**

### CODE REVIEW

File: Exiled.API/Features/Warhead.cs [hunk 1]
```diff
@@ -55,6 +57,49 @@ public static bool IsKeycardActivated
...
+        /// </summary>
+        public static WarheadStatus Status
+        {
+            get
+            {
+                if (!LeverStatus && !IsInProgress)
+                    return WarheadStatus.NotArmed;
+
+                if (LeverStatus && !IsInProgress)
+                    return WarheadStatus.Armed;
```
[the remark this answers is not in the document]
Comment (Michal78900) [reply Exiled.API/Features/Warhead.cs:71]: This doesn't make sense to me. The `Status` getter should return the most important thing.
The priority is Detonated > InProgress > Armed > NotArmed. Adding this check here would make `Status` return lever status instead InProgress for example.

[the remark this answers is not in the document]
Comment (louis1706) [reply Exiled.API/Features/Warhead.cs:71]: This should be the same ?
```
                if (!IsInProgress)
                {
                    if (!LeverStatus)
                        return WarheadStatus.NotArmed;
                    return WarheadStatus.Armed;
                }

                if (!IsDetonated)
                    return WarheadStatus.InProgress;

                return WarheadStatus.Detonated;

```

[the remark this answers is not in the document]
Comment (Michal78900) [reply Exiled.API/Features/Warhead.cs:71]: Now that you've added the `!` yes, it does make sense.

[the remark this answers is not in the document]
Comment (louis1706) [reply Exiled.API/Features/Warhead.cs]: Yes i have miss it

**Review by Ceglaa — APPROVED** [review]

**Review by NaoUnderscore — APPROVED** [review]

**Pull request merged**